pkg/util/metrics/util.go

```diff
@@ -75,3 +76,13 @@ func RegisterMetricAndTrackRateLimiterUsage(ownerName string, rateLimiter flowco
 	// }, updatePeriod, rateLimiterMetrics[ownerName].stopCh)
 	return nil
 }
+
+// SinceInMicroseconds gets the time since the specified start in microseconds.
+func SinceInMicroseconds(start time.Time) float64 {
+	return float64(time.Since(start).Nanoseconds() / time.Microsecond.Nanoseconds())
```

Comment:
This function has a terrible bug. https://play.golang.org/p/EuyaiMGHcxB

Reply:
This is at least the 2nd time I've pointed out a precision loss problem in metrics code. Any ideas on how we can test or otherwise prevent people from making this mistake?

Reply:
(I'm fine with the location unless @logicalhan has a better idea)

Reply:
We actually just purged the `pkg/util/metrics` directory. We've moved shared metrics into component-base (under `staging/src/k8s.io/component-base/metrics/prometheus`).

I think this PR will need to be rebased.

Reply:
Suggestion: add SinceInMicroseconds to the clock package, where this would be much more unit testable?

And don't resurrect this package which apparently has moved elsewhere :)

Reply:
+1 on @lavalamp's suggestion on moving stuff to clock.

Reply:
> This function has a terrible bug. https://play.golang.org/p/EuyaiMGHcxB

awesome finding. 

Seems there is no  `SinceInMicroseconds` anymore.  I don't know when it is deleted.